networkx/algorithms/centrality/betweenness_subset.py

```diff
@@ -187,11 +187,13 @@ def edge_betweenness_centrality_subset(G,sources,targets,
     return b
 
 # obsolete name
-def betweenness_centrality_source(G,normalized=True,weight=None,sources=None):
+def betweenness_centrality_source(G, normalized=True, weight=None,
+                                  sources=None):
     if sources is None:
-        sources=G.nodes()
-    targets=G.nodes()
-    return betweenness_centrality_subset(G,sources,targets,normalized,weight)
+        sources = list(G)
```

Comment:
@MridulS If you want, go ahead and make this change. Otherwise, we can save it for next time (but add it to your todo list).